Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get install -y \
 	python-pip
 	
 RUN pip install fabric --upgrade \
-	Requests --upgrade
+	pip install requests --upgrade \
```

Comment:
Adding `pip install`  at the beginning of each line makes them  different commands  instead of a single command spanning multiple lines